sdk/identity/identity/samples/AzureIdentityExamples.md

```diff
@@ -0,0 +1,351 @@
+
+## Authenticating with `DefaultAzureCredential`
+
+This example demonstrates authenticating the `SecretClient` from the [@azure/keyvault-secrets][secrets_client_library] client library using the `DefaultAzureCredential`. There's also [a runnable sample](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/identity/identity/samples/typescript/src/defaultAzureCredential.ts) to create a Key Vault key client you can copy-paste. The `DefaultAzureCredential` makes for a terrific starting point as it provides sane defaults with minimal configuration and chains multiple credentials together. While you may outgrow it eventually, it is a sensible first choice for most scenarios where the application is intended to ultimately be run in the Azure Cloud.
+
+```ts
+/**
+ * The default credential first checks environment variables for configuration.
+ * If environment configuration is incomplete, it will try managed identity.
+ */
+function withDefaultAzureCredential() {
+  const credential = new DefaultAzureCredential();
+  const client = new SecretClient(`https://key-vault-name.vault.azure.net`, credential);
+}
+```
+
+### Authenticating a user assigned managed identity with `DefaultAzureCredential`
+
+This example demonstrates authenticating the `SecretClient` from the [@azure/keyvault-secrets][secrets_client_library] client library using the `DefaultAzureCredential`, deployed to an Azure resource with a user assigned managed identity configured.
+
+For more information about how to configure a user assigned managed identity for an Azure resource please refer to [What are managed identities for Azure resources][azure_managed_identities].
+
+```ts
+/**
+ * The default credential will use the user assigned managed identity with the specified client ID.
+ */
+function withDefaultAzureCredential() {
+  // Alternatively, you may set the environment variable AZURE_CLIENT_ID="<MANAGED_IDENTITY_CLIENT_ID>" and omit the `managedIdentityClientId`
+  // option when using `DefaultAzureCredential` - the two approaches are equivalent.
+  const credential = new DefaultAzureCredential({
+    managedIdentityClientId: "<MANAGED_IDENTITY_CLIENT_ID>"
+  });
+  const client = new SecretClient("https://key-vault-name.vault.azure.net", credential);
+}
+```
+
+## Authenticating a user account interactively in the browser
+
+For clients that have a default browser available and for client-side applications running in the browser, the `InteractiveBrowserCredential` provides the simplest user authentication experience. In the sample below an application authenticates a `SecretClient` from the [@azure/keyvault-secrets][secrets_client_library] using the `InteractiveBrowserCredential`.
+
+> For client-side applications running in the browser, the `InteractiveBrowserCredential` is the only credential type that is supported.
```

Comment:
I'd like to call this sentence out - is this accurate?

Reply:
Yes!